src/main/java/com/flightstats/hub/config/properties/S3Properties.java

```diff
@@ -101,4 +101,6 @@ public int getMaxChunkInMB() {
         return propertiesLoader.getProperty("s3.maxChunkMB", 40);
     }
 
+    public long getLifecycleRulesManagerOffsetMinutes() { return propertiesLoader.getProperty("s3.lifecycleRulesManager.offset.minutes", 60 * 6); }
```

Comment:
The comment says the default is 660 in aws-hub.properties, but it looks here like the default is 360 (60*6)?

Reply:
Doh, thanks